### PR TITLE
ref(coinbase): list accounts and transactions, allows to deposit and withdraw funds

### PR DESCRIPTION
**Pull request by cmgustavo**
- Replaces testnet by sandbox using personal limited account
- Refactor code for getToken (async to Promises) and stored everything in the same key
- Refactor error messages
- Store user info when linked accounts
- Clean unused code to connect
- Removes buy/sell options
- Get all accounts from Coinbase (only wallet type)
- Filter accounts per BitPay supported coins
- UI: List accounts with icons and balance in Wallet tabs
- UI: Account details as wallet-details view
- New features: 
   - Deposit crypto from BitPay wallet into your Coinbase account
   - Withdraw crypto from your Coinbase account to BitPay Wallet

### CODE REVIEW

File: src/pages/integrations/coinbase/coinbase.html [hunk 1]
```diff
@@ -14,77 +14,26 @@
...
+          <img src="assets/img/coinbase/coinbase-logo.png" width="150">
         </expandable-header-primary>
       </ion-toolbar>
     </expandable-header>
 
-    <div *ngIf="!accessToken" class="center-header">
+    <div class="center-header">
       <div padding text-center>
-        <div *ngIf="!showOauthForm">Coinbase's exchange service is available in US, and can take 3-5 days to buy or sell bitcoin.</div>
+        <div *ngIf="!showOauthForm">Manage your Coinbase's accounts, check balance, deposit and withdrawal funds between wallets.</div>
```
Comment (bsnowden3) [inline src/pages/integrations/coinbase/coinbase.html]: I would change this sentence to "Manage your Coinbase account, check balances, deposit and withdraw funds between wallets."

File: src/pages/home/home.ts [hunk 2]
```diff
@@ -73,6 +74,16 @@ export class HomePage {
       /* imgSrc: TODO 'assets/img/bitpay-card-solid.svg' */
       imgSrc: 'assets/img/icon-bpcard.svg'
     },
+    {
+      name: 'coinbase',
+      title: 'Coinbase',
+      body: 'Deposit and withdrawal with your coinbase account.',
```
Comment (bsnowden3) [inline src/pages/home/home.ts]: "Deposit and withdraw with your coinbase account"

File: src/pages/wallets/wallets.html [hunk 3]
```diff
@@ -99,6 +99,60 @@
...
+            <div action-card-header>
+              <div class="zero-state-coinbase">
+                <div class="zero-state-image">
+                  <img src="assets/img/coinbase/coinbase-logo-white.svg">
+                </div>
+              </div>
+            </div>
+            <div action-card-title translate>Connect Account</div>
+            <div action-card-body translate>
+              Deposit and withdrawal with your coinbase account.
```
[the remark this answers is not in the document]
Comment (cmgustavo) [reply src/pages/wallets/wallets.html]: Fixed. Thanks

File: src/pages/integrations/coinbase/coinbase.html (hunk 1 above)
Comment (cmgustavo) [reply src/pages/integrations/coinbase/coinbase.html]: Fixed!

File: src/pages/home/home.ts (hunk 2 above)
Comment (cmgustavo) [reply src/pages/home/home.ts]: done

**Pull request merged**